### DISCUSSION TITLE
Fullscreen mode for pavucontrol-qt

### DISCUSSION THREAD

**Discussion by waarmond** [lxqt/lxqt#2565] [General]
With my desktop resolution and DPI settings https://github.com/lxqt/pavucontrol-qt may only show ~11 sound-producing apps per screen. If you have more the most recently started ones have to be reached by scrolling.

On my desktop environment a window much higher than the vertical pixels of my screen is created when there are >11 open apps thus I have to resize the pavucontrol window before being able to scroll.

In https://github.com/lxqt/pavucontrol-qt/blob/master/src/mainwindow.ui I only find
```
<width>766</width>
<height>548</height>
```
Where is the option allowing/denying window maximization?

**Comment by tsujan** [comment]
> Where is the option allowing/denying window maximization?

The window can be maximized like any other window. It's done by the WM's title-bar.
